lieer/gmailieer.py

```diff
@@ -191,6 +202,7 @@ def setup (self, args, dry_run = False, load = False):
     self.dry_run          = dry_run
     self.HAS_TQDM         = (not args.no_progress)
     self.credentials_file = args.credentials
+    self.age = args.age if hasattr(args, 'age') else None
```

Comment:
Can this be just `args.age`?

Reply:
I'm not sure if it make sense to use age parameter for push action. For now args.age doesn't exist for push action.

Reply:
I think the parameter should only be available on the `set` command. Having the above line in def setup() and in the action as well is redundant, it should only be on the action unless it is a command argument.

Reply:
As I review your last feedback, I found only messages.list and threds.list method support this query. According to https://developers.google.com/gmail/api/guides/filtering,
> You can search or filter files using the messages.list and threads.list methods. These methods accept the q parameter which supports the same advanced search syntax as the Gmail web-interface. 

So args.age doesn't make sense for push action.